alien4cloud/docker-alien4cloud
# Added Dockerfile and hooks

# Pull Request description

## Description of the change

Added Dockerfile and hooks that will be used as source for an automated build on docker hub.

### How to verify it

See example of similar work at https://github.com/laurentganne/docker-alien4cloud and corresponding automated build at https://hub.docker.com/r/laurentg/docker-alien4cloud/
